src/axom/primal/operators/detail/intersect_impl.hpp

```diff
@@ -1856,7 +1860,7 @@ inline bool intersect_line_bilinear_patch(const Line<double, 3>& line,
             if(t1 == t2)
             {
               t.push_back(0.5 * t1);
-              v.push_back(0.5);
+              u.push_back(0.5);
```

Comment:
This line looked like a cut/paste error considering the rest of the code block pushing values into the ``u`` vector.

Reply:
Based on the surrounding code, I agree.

@jcs15c -- please check this and confirm.
Also, can we come up with a test case that would catch this regression?

Reply:
Sorry to leave you hanging! Yes, this should definitely be a push to the `u` vector. I'll look into finding these issues more systematically with testing.

Reply:
@jcs15c - could I get an approval on this PR then?